nativelib/src/main/scala/scala/scalanative/unsafe/resolvedAtLinktime.scala

```diff
@@ -0,0 +1,11 @@
+package scala.scalanative.unsafe
+
+import scala.annotation.StaticAnnotation
+import scala.annotation.meta.{field, getter}
+import scala.scalanative.runtime.intrinsic
+
+/** Used to annotate that given value should be resolved at link-time,
+ *  based on provided `withName` parameter or on fully-qualified name otherwise */
+@field @getter
+private[scalanative] class resolvedAtLinktime(withName: String = intrinsic)
```

Comment:
Consider making the parameter mandatory. This annotation should be used rarely enough that it is not worth trying to guess what a good default should be.
```suggestion
private[scalanative] class resolvedAtLinktime(withName: String)
```